runner/app/live/pipelines/segment_anything_2.py

```diff
@@ -0,0 +1,105 @@
+import io
+import logging
+import threading
+import time
+from typing import List, Optional
+import cv2
+import numpy as np
+from PIL import Image
+from pydantic import BaseModel
+from Sam2Wrapper import Sam2Wrapper
+from .interface import Pipeline
+
+logger = logging.getLogger(__name__)
+
+class Sam2LiveParams(BaseModel):
+    class Config:
+        extra = "forbid"
+
+    model_id: str = "facebook/sam2-hiera-tiny"
+    point_coords: List[List[int]] = [[1, 1]]
+    point_labels: List[int] = [1]
+    show_points: Optional[bool] = True
```

Comment:
We don't need the `Optional` when there's a default value! So we can fix the typing here in case it's complaining when using it directly.
```suggestion
    show_points: bool = True
```